kpatch-build/create-klp-module.c

```diff
@@ -208,8 +239,23 @@ static void create_klp_relasecs_and_syms(struct kpatch_elf *kelf, struct section
 		ALLOC_LINK(rela, &klp_relasec->relas);
 		rela->sym = sym;
 		rela->type = krelas[index].type;
-		rela->offset = krelas[index].offset;
-		rela->addend = krelas[index].addend;
+		if (!strcmp(base->name, ".toc"))
+			rela->offset = toc_offset;
+		else
+			rela->offset = krelas[index].offset;
```

Comment:
I saw the comment, I think I just didn't fully understand it.  Now it makes sense.  No need to move the comment unless you want to.